src/append/appendArrayBuffer.ts

```diff
@@ -29,11 +29,13 @@ export async function appendArrayBuffer(
 async function getExtendedSourceFromArrayBuffer(
   originalRelativePath: string,
   arrayBuffer: SupportedBufferInput,
-  options: { dateModified?: number } = {},
+  options: { dateModified?: number; extra?: boolean } = {},
 ): Promise<ExtendedSourceItem> {
   const { name, relativePath } = getNameInfo(originalRelativePath);
   const blobInput = await arrayBuffer;
-  // @ts-expect-error Blob doesn't explicitly accept ArrayBufferLike but that should work.
+
+  // Types of property [Symbol.toStringTag] are incompatible.
+  // @ts-expect-error Type SharedArrayBuffer is not assignable to type ArrayBuffer
```

Comment:
It seems you prefer to comment with what error is silenced while I prefer to explain why. Maybe we should combine both?